From 8a4292268c2d88cdf757b4c162315360901139c5 Mon Sep 17 00:00:00 2001
From: Andreas Mayer <andreas.mayer@steffing.de>
Date: Fri, 29 Dec 2017 21:09:36 +0100
Subject: Fix SEPA plugin installation

Summary:
add_library() may add a "lib" prefix to the plugin's filename, which causes it not to be found.
Using kde4_add_plugin() instead again fixes that.

This change to add_library() was (accidentally?) backported from the KF5 version in commit c78183f18201 .

BUG: 386547
FIXED-IN: 4.8.2

Intended for the 4.8 branch only, of course.

Test Plan:
The plugin is again installed as konlinetasks_sepa.so here (like in 4.8.0, and also the KF5 port).
The error message that the plugin could not be found does not occur anymore, kmymoney prints

> onlineTask available "org.kmymoney.creditTransfer.sepa"

 to stdout now.

Haven't tested the actual functionality though.

Reviewers: #kmymoney, habacker

Reviewed By: #kmymoney, habacker

Subscribers: tbaumgart

Tags: #kmymoney

Differential Revision: https://phabricator.kde.org/D9545
---
 kmymoney/plugins/onlinetasks/sepa/CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/kmymoney/plugins/onlinetasks/sepa/CMakeLists.txt b/kmymoney/plugins/onlinetasks/sepa/CMakeLists.txt
index e3962d7..031820e 100644
--- kmymoney/plugins/onlinetasks/sepa/CMakeLists.txt
+++ kmymoney/plugins/onlinetasks/sepa/CMakeLists.txt
@@ -13,7 +13,7 @@ kde4_add_ui_files( sepaOnlineTasks_SRCS
   ui/sepacredittransferedit.ui
 )
 
-add_library(konlinetasks_sepa MODULE
+kde4_add_plugin(konlinetasks_sepa
   ${sepaOnlineTasks_SRCS}
 )
 
-- 
cgit v0.11.2

